api/serializers.py

```diff
@@ -35,8 +39,46 @@ class Meta:
         model = User
         fields = ("token", "username", "password")
 
-
 class VideoSerializer(serializers.ModelSerializer):
+    def create(self, validated_data):
+        data = self.context.get("data")
+        sender = self.context.get("user")
+
+        # receiver = User.objects.get(id=validated_data.pop('receiver'))
+        # Data is prepended by "data:video/mp4;base64,", need to remove those 22 characters
+        # Encode converts from string to bytes
+        data_bytes = data[22:].encode("ascii")

```

Comment:
We discussed this but for the purpose of just having files uploading this is fine. But I don't see why we are stripping the scheme part of the data URL. You may as well just leave it there and avoid having to tack it on later.

Reply:
Yup this is silly

Reply:
Unclear if you are going to leave in the removal of the 22 chars. I don't understand why they need to be removed only to be added in again later.
```suggestion
        data_bytes = data.encode()
```